src/Components/TextBlockWithLabel.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, ReactElement, isValidElement } from "react";
+import styled from "styled-components";
+import { P8, P5/*, Stack*/ } from "@deskpro/app-sdk";
+
+export type Props = {
+    label?: string | ReactElement,
+    text?: string | number | ReactElement,
+    marginBottom?: number,
+};
+
+const Container = styled.div<Props>`
+    margin-bottom: ${({ marginBottom }) => `${marginBottom}px`};
+`;
+
+const Label = styled(P8)`
+    color: ${({ theme }) => (theme.colors.grey80)};
+`;
+
+const TextBlockWithLabel: FC<Props> = ({ text, label, marginBottom = 10 }) => {
```

Comment:
When I was creating stories in the `@deskpro/app-sdk`, I noticed that this is a clone of the `Property` component. Try using it, it might work for you
```js
import { Property } from "@deskpro/app-sdk";
```

Reply:
Is that component being exported? Or in what version is it? Property is a type here

Reply:
I don't know when it was added, but I'm pretty sure it's available in your version `"@deskpro/app-sdk": "^1.2"`